pkg/api/validation/validation.go

```diff
@@ -671,6 +679,25 @@ func validateNFSVolumeSource(nfs *api.NFSVolumeSource, fldPath *field.Path) fiel
 	return allErrs
 }
 
+func validateQuobyteVolumeSource(quobyte *api.QuobyteVolumeSource, fldPath *field.Path) field.ErrorList {
+	glog.V(4).Infof("quobyte vol %#v registry %#v", quobyte.Volume, quobyte.Registry)
+	allErrs := field.ErrorList{}
+	if len(quobyte.Registry) == 0 {
+		allErrs = append(allErrs, field.Required(fldPath.Child("registry"), "Quobyte registry can't be nil"))
+	} else {
+		for _, hostPortPair := range strings.Split(quobyte.Registry, ",") {
+			if _, _, err := net.SplitHostPort(hostPortPair); err != nil {
+				allErrs = append(allErrs, field.Required(fldPath.Child("registry"), fmt.Sprintf("Quobyte registry %v", err)))
```

Comment:
error details should be imperative fragments: "must be foo" or "may not bar" or similar.   The field name will already be present in the error, so "Quobyte registry" is redundant.  Printing an error here makes this fairly opaque - can you just type out the error youw ant to see?